docs/indicators/domain-extraction.md

```diff
@@ -0,0 +1,57 @@
+---
+id: domain-extraction
+title: Domain Extraction
+---
+
+The Cortex XSOAR domain indicator type is built using regular expression and a formatting script.
+The following describes the domain extraction components and what output you should expect when extracting indicators of type domain.
+
+## Domain Extraction Components
+
+There are two components when extracting domain indicators:
+- Regular expression
+- Formatting script
+
+### Regular Expression
+
+When text is given, a domain regular expression will try to catch a valid domain based on the following characteristics:
+- A domain with ASCII and non-ASCII characters.
+- Escaped and unescaped domains.
+
+The regular expression can extract domains from one of the following:
+- Explicit domain.
+- URL.
+- Email address.
+
+### Formatting Script
+
+After extracting the domain using a regular expression, an **ExtractDomainAndFQDNFromUrlAndEmail** formatting script iterates on each given domain and does the following:
+
+1. Replaces "[.]" with ".".
+	
+	For example:
+ 
+	www[.]evil.com --> www.evil.com
```

Comment:
@ShirleyDenkberg I saw you removed the '' from the www.evil.com and now it's a hyperlink. Like other from the Common Domain Structures section.

It's a problem, so either we find a way to remove the hyperlink or keep the ''.

Reply:
![image](https://user-images.githubusercontent.com/61933087/149144228-ccd54192-02c8-410f-b28c-061e60cd2ed8.png)